### PR TITLE
Adding testnet node/wallet guide

### PR DESCRIPTION
**Pull request by kehiy**

### CODE REVIEW

File: website/_i18n/en.yml [hunk 1]
```diff
@@ -450,6 +450,7 @@ dict:
     run_pactus_docker: How to run Pactus using Docker?
     send_transactions_gui: How to send transactions in Graphic Mode?
     use_wallet_cli: How to use wallet in Command Line Mode?
+    pactus_testnet: How to use and interact with Pactus testnet (Pheonix)?
```
Comment (b00f) [inline website/_i18n/en.yml:453]:
```suggestion
    pactus_testnet: How to use and interact with Pactus Testnet (Pheonix)?
```

**Review by b00f — APPROVED** [review]

File: website/_i18n/en.yml [hunk 2]
```diff
@@ -450,6 +450,7 @@ dict:
     run_pactus_docker: How to run Pactus using Docker?
     send_transactions_gui: How to send transactions in Graphic Mode?
     use_wallet_cli: How to use wallet in Command Line Mode?
+    pactus_testnet: How to use and interact with Pactus Testnet (Pheonix)?
```
Comment (Mr-HoDL58) [inline website/_i18n/en.yml:453]: Phoenix

File: website/_i18n/en/user-guides/pactus-testnet/index.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
+---
+version: 8
+---
+
+## Preface
+
+Pactus Blockchain has a Test network for developers to test their software which is using Pactus, users
```
Comment (Mr-HoDL58) [inline website/_i18n/en/user-guides/pactus-testnet/index.md:7]: Pactus Blockchain has introduced a Test network called Phoenix for developers and users to test their software utilizing Pactus technology. This guide will demonstrate how to connect and interact with Phoenix using official Pactus software designed for the main network.

File: website/_i18n/en/user-guides/pactus-testnet/index.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+To get started, download and extract the Pactus software
+[here]({{ site.baseurl }}/download).
+Once you have downloaded and extracted the Pactus software file, you can proceed to the next step.
+
+---
+
+## Phoenix network
+
+Pactus has 3 different genesis block types for mainnet, testnet and localnet (used for setting up
```
Comment (Mr-HoDL58) [inline website/_i18n/en/user-guides/pactus-testnet/index.md:22]: Pactus offers three distinct genesis block types for mainnet, testnet, and localnet (used for developing a dev environment). The genesis type is defined within your Pactus working directory and wallet file once it has been created or initialized. Subsequent commands will automatically determine the network type based on this information, eliminating the need to specify the network being used.

File: website/_i18n/en/user-guides/pactus-testnet/index.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+dev environment).  this genesis type is determined in your Pactus working directory and your wallet
+file once you made or initialized it. the rest of commands can find-out the network type based on
+this and there is no need to specify which network you are using.
+
+## CLI Softwares
+
+To interact with pheonix using wallet, daemon CLI, there is a flag called `--testnet` provided on
```
Comment (Mr-HoDL58) [inline website/_i18n/en/user-guides/pactus-testnet/index.md:29]: Phoenix

File: website/_i18n/en/user-guides/pactus-testnet/index.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+{% endtab %}
+
+{% endtabs %}
+
+You can use this flag for recover command as well.
+
+{% alert_start info %}
+<b>Wallet Address</b>
+<br><br>
+Pactus address have pc HRP on mainnet and a tpc HRP on testnet. any address with pc prefix is invalid
```
Comment (Mr-HoDL58) [inline website/_i18n/en/user-guides/pactus-testnet/index.md:59]: Pactus addresses have pc HRP on Mainnet and a tpc HRP on testnet. Addresses with pc prefix are invalid on Testnet.

File: website/_i18n/en/user-guides/pactus-testnet/index.md [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+
+## Download
+
+To get started, download and extract the Pactus software
+[here]({{ site.baseurl }}/download).
+Once you have downloaded and extracted the Pactus software file, you can proceed to the next step.
+
+---
+
+## Phoenix network
```
Comment (Mr-HoDL58) [inline website/_i18n/en/user-guides/pactus-testnet/index.md:20]: Phoenic Network

File: website/_i18n/en/user-guides/pactus-testnet/index.md [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+{% alert_start info %}
+<b>Wallet Address</b>
+<br><br>
+Pactus address have pc HRP on mainnet and a tpc HRP on testnet. any address with pc prefix is invalid
+on testnet.
+<br><br>
+You can check pip-8 for more details on Pactus address.
+{% alert_end %}
+
+## GUI Node and wallet
```
Comment (Mr-HoDL58) [inline website/_i18n/en/user-guides/pactus-testnet/index.md:65]: Wallet

File: website/_i18n/en/user-guides/pactus-testnet/index.md [hunk 9]
```diff
@@ -0,0 +1,96 @@
...
+
+Pactus also offers a GUI to run a node, if you want to use GUI as a testnet node you have to run it
+wit the same flag in your terminal.
```
Comment (Mr-HoDL58) [inline website/_i18n/en/user-guides/pactus-testnet/index.md:68]: with

File: website/_i18n/en/user-guides/pactus-testnet/index.md [hunk 10]
```diff
@@ -0,0 +1,96 @@
...
+```
+
+{% endtab %}
+
+{% endtabs %}
+
+## Notes
+
+As it's mentioned, the rest of work will be handled by the software which read the network type from
+wallet file and working directory. the rest of commands and instructions can be found in mainnet docs.
```
Comment (Mr-HoDL58) [inline website/_i18n/en/user-guides/pactus-testnet/index.md:93]: The rest of the commands

File: website/_i18n/en/user-guides/pactus-testnet/index.md [hunk 11]
```diff
@@ -0,0 +1,96 @@
...
+
+When you make multiple wallet and nodes, may you get conflicts with old wallet file or working directories.
```
Comment (Mr-HoDL58) [inline website/_i18n/en/user-guides/pactus-testnet/index.md:95]: wallets

File: website/_i18n/en/user-guides/pactus-testnet/index.md [hunk 12]
```diff
@@ -0,0 +1,96 @@
...
+you have to specify a new path for them using `--path` flag or just remove the old one.
```
Comment (Mr-HoDL58) [inline website/_i18n/en/user-guides/pactus-testnet/index.md:96]: You

File: website/_i18n/zh.yml [hunk 13]
```diff
@@ -419,6 +419,7 @@ dict:
     run_pactus_docker: 如何使用Docker运行帕图斯？
     send_transactions_gui: 如何在图形模式下发送交易？
     use_wallet_cli: 如何在命令行模式下使用钱包？
+    pactus_testnet: 如何使用 Pactus 测试网（Pheonix）并与之交互？
```
Comment (Mr-HoDL58) [inline website/_i18n/zh.yml:422]: Phoenix

**Comment by Mr-HoDL58** [comment]
bnroken links fail also.

**Comment by kehiy** [comment]
@Mr-HoDL58 requested changes addressed.

**Pull request merged**